feature-account-api/src/main/java/jp/co/soramitsu/feature_account_api/domain/model/TotalBalance.kt

```diff
@@ -0,0 +1,9 @@
+package jp.co.soramitsu.feature_account_api.domain.model
+
+import java.math.BigDecimal
+
+data class TotalBalance(val balance: BigDecimal, val fiatSymbol: String) {
+    companion object {
+        val Empty = TotalBalance(BigDecimal.ZERO, "$")
```

Comment:
maybe "$" should be defined as const somewhere

Reply:
agree, will fix